Fix T970871 and remove encodeHtml from docs (#2180)
DevExpress/devextreme-documentation: concepts/Common/Security Considerations/20 HTML Encoding/20 Encoding Across Components/messageHtml.md

[DevExtreme Dialog UI methods](/api-reference/50%20Common/utils/ui/dialog/alert(messageHtml_title).md '/Documentation/ApiReference/Common/Utils/ui/dialog/#alertmessageHtml_title') accept an unencoded HTML string as a dialog message. Encode this string in the following manner:

    <!-- tab: JavaScript -->
    $(function() {

>         // Use your favorite sanitizing tool to encode the `message` string:
>         const message = "Are you sure?<script>alert('XSS')</script>";

Reviewer: ```suggestion
        const message = "Are you sure?<script>alert('XSS')</script>";
        // ...
        // Encode the `message` string with your favorite sanitizing tool
        // ...
```

[DevExtreme Dialog UI methods](/api-reference/50%20Common/utils/ui/dialog/alert(messageHtml_title).md '/Documentation/ApiReference/Common/Utils/ui/dialog/#alertmessageHtml_title') accept an unencoded HTML string as a dialog message. Encode this string in the following manner:

    <!-- tab: JavaScript -->
    $(function() {
        const message = "Are you sure?<script>alert('XSS')</script>";
        // ...
        // Encode the `message` string with your favorite sanitizing tool
        // ...
        DevExpress.ui.dialog.confirm(message, "Confirm changes");

>     });

Reviewer: This samples doesn't require mentions of jQuery.

```suggestion
    const message = "Are you sure?<script>alert('XSS')</script>";
    // ...
    // Encode the `message` string with your favorite sanitizing tool
    // ...
    DevExpress.ui.dialog.confirm(message, "Confirm changes");
```
